calico-cloud/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -2,7 +2,7 @@
 description: Learn about policies, tiers, and policy evaluation.
 ---
 
-# Understanding policy tiers
+# Policy tiers tutorial
```

Comment:
The others are renamed "Get started with policy". Should this one be the same?

Reply:
This is because CC uses a different taxonomy for tutorials. Eventually, we should align tutorial titles across products.